### ISSUE TITLE
Zite request: Fan art hosting zite

### ISSUE THREAD

**Issue by FlashFried** [HelloZeroNet/ZeroNet#2802]
Decentralization is an amazing thing. One thing I'd like to suggest is that there be a clonable template for creating fan art sites. There are plenty of people who have wanted to establish an alternative to centralized fan art sites.

**Comment by sfsfrgrg** [comment]
yes,butYes, but it has stopped updating and it is estimated that it will not reply to your questions

**Comment by sfsfrgrg** [comment]
Yes, but it has stopped updating and it is estimated that it will not reply to your questions

**Comment by caryoscelus** [comment]
why not use 0gallery (or 1+gallery)?
